i18n-guidelines.md

```diff
@@ -1,24 +1,96 @@
-# 🌐 i18n Guide
+# 🌐 i18n Guidelines
 
-At the moment, we have the course in the following languages:  
+Welcome to our i18n Guidelines! We appreciate your interest in translating our Intro to Open Source course.
 
-* Brazilian Portuguese
-* English
-* French
+At the moment, we have the course in the following languages:
 
-## How can I contribute?
+- English
+- [French](https://intro.opensauced.pizza/#/translations/fr/README)
+- Brazilian Portuguese — work in progress
 
-* **Translate our README**. There are two methods of translation contributions we accept.
-  * **Official Translations**: Official translations start as a post in our discussion board. If there is enough interest and volunteers, we can add the official translation as an option to view within our README.
-  * **Community Translations**: We can’t always support the maintenance of translations. However, we do know that there are contributors out there willing to translate. For that reason, we have our Community Translations section. If you’ve forked the repository and created a translation, you can add a link to your forked translation to `community-translations.md`. We encourage you to also add it to the discussion board in the event it becomes popular enough and has enough support to be moved to an official translation.
-* **Review PRs and translations.** New to reviewing GitHub PRs and translations? No worries, we have you covered in our [Reviewer Process Guide](#reviewer-process-guide).
+## How Can I Contribute?
+
+There are two types of i18n contributions that we accept:
+
+1. Translate our Intro to Open Source course.
+2. Review pull requests (PRs) and translations.
+
+### Translate the Intro to Open Source Course
+
+We have two types of translations:
+
+#### 1. Official Translations
+
+Official translations start as a post on our [discussion board](https://github.com/open-sauced/intro/discussions). If there is enough interest and volunteers, we can add the official translation as an option to view within our README.
+
+#### 2. Community Translations
+
+We can’t always support the maintenance of translations. However, we know that some contributors are willing to translate. For that reason, we have our Community Translations section. If you've forked the repository and created a translation, you can add a link to your forked translation to the `community-translations.md` file.
```

Comment:
Do we now have the Community Translation section? Or do we still need to create this section?

This part is not clear yet as in how to create a community translation when folks are interested to translate our course outside our official translations.

> If you've forked the repository and created a translation ...

It might have been clearer if we put it this way (if this is the intention):

> If you're interested in translating our Intro to OSS course, fork this repo and add the translation in you forked repo. Then you you can add a link to your forked translation to the `community-translations.md` file.

Reply:
At the moment, we don't. @BekahHW what do you think about this?

Reply:
@BekahHW is this related to issue #71?

That we create a `community-translations.md` with content as below:

```markdown
# Community Course Translations

We are grateful for the contributions of the community in translating this course. Although the OpenSauced team does not currently maintain these projects, we want to share the forked copy of the repositories with those taking the course. Below, you will find the latest community translations.

## Community Translations

- [Translated Language](link to the forked repo)
```